### ISSUE TITLE
No CPU!

### ISSUE THREAD

**Issue by LonelyGuy12** [AlexFlipnote/neofetch-win#19]
```C:\Users\phoen>neofetch
################   ################      phoen@Lonely
################   ################      ------------
################   ################      OS: Windows 11 Home Insider Preview Single Language
################   ################      Uptime: 2 hours, 31 minutes and 9 seconds
################   ################      Local IP: 192.168.1.45
################   ################      Motherboard: ASUSTeK COMPUTER INC. (X513IA)
################   ################      CPU: Not found...??
                                         GPU: AMD Radeon(TM) Graphics
################   ################      Memory: 5.5GiB / 7.4GiB (74.75%)
################   ################      Disk: C: 127.8GiB / 237.2GiB (53.88%)
################   ################            D: 69.2GiB / 931.5GiB (7.43%)
################   ################
################   ################
################   ################
################   ################
```

CPU Not Found!
I own a Ryzen 7 4700u

**Comment by LonelyGuy12** [comment]
BTW don't tell me I leaked my IP. That is a random IP

**Comment by AlexFlipnote** [comment]
Try again now and see if it works

**Comment by sadorowo** [comment]
```
################   ################      Sador@DESKTOP-LQMT0BK
################   ################      ---------------------
################   ################      OS: Windows 11 Pro
################   ################      Uptime: 2 hours, 15 minutes and 0 seconds
################   ################      Local IP: [HIDDEN]
################   ################      Motherboard: ASRock (H97M Pro4)
################   ################      CPU: Not found...??
                                         GPU: NVIDIA GeForce GTX 970
################   ################      Memory: 6.5GiB / 15.9GiB (40.84%)
################   ################      Disk: C: 248.9GiB / 446.4GiB (55.74%)
################   ################
################   ################
################   ################
################   ################
################   ################
```
same error, not fixed

**Comment by AlexFlipnote** [comment]
Can you open PowerShell and do the following command?
`Get-WmiObject -Class Win32_Processor -ComputerName. | Select-Object -Property name > test.txt`
Then upload the txt file to me somewhere so I can see what is going on with raw data?

**Comment by LonelyGuy12** [comment]
> Can you open PowerShell and do the following command? `Get-WmiObject -Class Win32_Processor -ComputerName. | Select-Object -Property name > test.txt` Then upload the txt file to me somewhere so I can see what is going on with raw data?

[test.txt](https://github.com/AlexFlipnote/neofetch-win/files/8099050/test.txt)

**Comment by LonelyGuy12** [comment]
It gives the proper CPU details in the txt file

**Comment by AlexFlipnote** [comment]
Just released 1.2.2 out now, hopefully that fixes the bug.
Update me on if it did.

**Comment by sadorowo** [comment]
on my pc still not fixed, i updated this a few minutes ago

**Comment by sadorowo** [comment]
> Can you open PowerShell and do the following command? `Get-WmiObject -Class Win32_Processor -ComputerName. | Select-Object -Property name > test.txt` Then upload the txt file to me somewhere so I can see what is going on with raw data?

this command returns:
```

name                                   
----                                   
Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz

```

**Comment by AlexFlipnote** [comment]
> > Can you open PowerShell and do the following command? `Get-WmiObject -Class Win32_Processor -ComputerName. | Select-Object -Property name > test.txt` Then upload the txt file to me somewhere so I can see what is going on with raw data?
> 
> this command returns:
> 
> ```
> 
> name                                   
> ----                                   
> Intel(R) Core(TM) i5-4590 CPU @ 3.30GHz
> ```

Could you also upload a .txt file so I can see why the issue still remains?

**Comment by sadorowo** [comment]
[test.txt](https://github.com/AlexFlipnote/neofetch-win/files/8117689/test.txt)

**Comment by AlexFlipnote** [comment]
Update to `1.2.3` see if it works, does for me at least.
![image](https://user-images.githubusercontent.com/19587382/155278915-67f2dfd4-e84c-4d70-8d6c-6ae6a3848bb5.png)

**Comment by sadorowo** [comment]
![image](https://user-images.githubusercontent.com/97941280/155337255-15207470-6fd0-427d-9e0c-f80aae8dbb9f.png)

**Comment by nounouthereal** [comment]
m

**Comment by AlexFlipnote** [comment]
I honestly have 0 clue on how to fix this then.. are you able to run a local copy of the code and find the issue? Since I do not have access to your PC.

**Comment by reidlabb** [comment]
![image](https://user-images.githubusercontent.com/45904911/156902439-2e090f7a-d913-4e08-8ee5-81ac3bcd8aa9.png)
same thing is happening for me, and the `Get-WmiObject -Class Win32_Processor -ComputerName.` command says its name is : Intel(R) Core(TM) 15-6500 CPU @ 3.20GHz

**Comment by Xaliks** [comment]
![1UA13](https://user-images.githubusercontent.com/64866197/157506130-5422e33c-8eca-4ddd-b6cc-0ff0ae47921a.png)

`Get-WmiObject -Class Win32_Processor -ComputerName. | Select-Object -Property name > test.txt`:
```
name                                           
----                                           
AMD Ryzen 5 3600 6-Core Processor              
```

**Comment by SimPilotAdamT** [comment]
Same issue for me...
![image](https://user-images.githubusercontent.com/71192298/169667210-1c3bae39-0ce2-4757-ab1f-857897066717.png)
Also go the GPU thing as well. Happens when multiple GPU's are detected.

**Comment by AlexFlipnote** [comment]
This should be fixed in 1.3.2 coming to PyPi pretty soon
